homeassistant/components/device_tracker/proximity.py

```diff
@@ -0,0 +1,279 @@
+"""
+custom_components.proximity
+~~~~~~~~~~~~~~~~~~~~~~~~~
+Component to monitor the proximity of devices to a particular zone. The result is an entity created in HA which maintains the proximity data
+
+Use configuration.yaml to enable the user to easily tune a number of settings:
+- Zone: the zone to which this component is measuring the distance to. Default is the home zone
+- Override Zones: where proximity is not calculated (e.g. work or school)
```

Comment:
Shouldn't this be called ignore zones ? Because they don't override anything in this component.